cfpb/capital-framework
# Use CFPB brand colors in cf-icons component

Adds CFPB brand colors as the default variable values for colors in `cf-icons`. 


## Testing

In this repo:
- check out this branch
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- Compare colors on local site: 
   - http://localhost:3000/components/cf-icons/ 
   - to the live cf documentation to make sure they match: https://cfpb.github.io/capital-framework/components/cf-icons/


## Notes

-

## Todos

-

## Checklist
